### PR TITLE
event: Page for Octobers Hacktoberfest meetings

### PR DESCRIPTION
**Pull request by ns-rse**
Event details and overview. I think a page for each week would be overkill.

I've included the resources available from [registering our event](https://hacktoberfest.com/events/#register) and have so far registered the first event but will do the rest tomorrow.

Still need to do all the announcements on Meetup and the 74 other mailing lists! :clown_face:

### CODE REVIEW

File: posts/2024-10/index.qmd [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+layout: post
+title:  "SheffieldR : Hacktoberfest 2024"
+date:   2024-10-04 17:30:00 +0000
+categories: meetups
+image: img/DO-HFest-EmailBanner-600px-2-@2x.png
+excerpt_separator: <!--more-->
+---
+
+
+We kick off this semesters Sheffield R User Group meetings in October with a series of hackathons to participate in
```
Comment (tdjames1) [inline posts/2024-10/index.qmd]:
```suggestion
We kick off this semester's Sheffield R User Group meetings in October with a series of hackathons to participate in
```

File: posts/2024-10/index.qmd [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+![Hacktoberfest 2024](img/DO-HFest-EmailBanner-600px-2-@2x.png)
+
+Hacktoberfest is an annual "_month-long celebration of all things open-source_" that runs across [GitHub][gh] and
+[GitLab][gl] which "_welcome[s] anyone who is interested in diving in, and everyone who's already part of the open-source
+software community._".
+
+We have planned four meetings across the month of October to help and support each other in contributing to open-source
+projects related to the [R][r] language and ecosystem including [rOpenSci][ropensci] and
+[Bioconductor][bioconductor]. The first session on Friday 2024-10-04 we will go over the basics of Git and contributing
```
Comment (tdjames1) [inline posts/2024-10/index.qmd]:
```suggestion
[Bioconductor][bioconductor]. In the first session on Friday 2024-10-04 we will go over the basics of Git and contributing
```

File: posts/2024-10/index.qmd [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+via [GitHub][gh]/[GitLab][gl] to open-source projects and how to go about finding R projects with issues that you could
+tackle and solve as part of Hacktoberfest 2024.
+
+The remaining sessions will be held weekly on Fridays at lunch time from 12:30-13:30 in the University of Sheffield's
+Students Union you can come along and seek support and offer help to those who have questions, whether that is with some
```
Comment (tdjames1) [inline posts/2024-10/index.qmd]:
```suggestion
Students Union. During these sessions you can come along and seek support and offer help to those who have questions, whether that is with some
```

File: posts/2024-10/index.qmd [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Preparation
+
+To participate you will need a [GitHub][gh] and/or a [GitLab][gl] account and to register between 2024-09-23 and
+2024-10-31 on the [Hacktoberfest site][hack24participation].
+
+## Learn More
+
+There are a bunch of resources available at [Resources for Beginners][hack24beginners] if you are unfamiliar with
+[Git][git], [GitHub][gh]/[GitLab][gl] but these aren't essential as we are aiming to cover the basics and help you over
```
Comment (tdjames1) [inline posts/2024-10/index.qmd]:
```suggestion
[Git][git] or [GitHub][gh]/[GitLab][gl] but these aren't essential as we are aiming to cover the basics and help you over
```

File: posts/2024-10/index.qmd [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+the course of these events.
+
+If you are interested in making contributions to R core checkout the information from [R Contributor
```
Comment (tdjames1) [inline posts/2024-10/index.qmd]:
```suggestion
If you are interested in making contributions to R core - the code and documentation that gets distributed as base R - check out the information from [R Contributor
```

**Review by tdjames1 — APPROVED** [review]
Thanks for setting up the event page, @ns-rse. I've made a few small suggestions to wording. I wonder if it's worth putting something about the sessions being open to all?

Is it necessary to include all of the hacktoberfest logos?

**Comment by ns-rse** [comment]
P.S. I stuck all the logos and material in so they could be reviewed for alternatives and also available for making some session slides.

**Pull request merged**